_overviews/scala3-book/taste-collections.md

```diff
@@ -9,17 +9,16 @@ next-page: taste-contextual-abstractions
 ---
 
 
-
 The Scala library has a rich set of collection classes, and those classes have a rich set of methods.
 Collections classes are available in both immutable and mutable forms.
 
-
-
 ## Creating lists
 
 To give you a taste of how these work, here are some examples that use the `List` class, which is an immutable, linked-list class.
 These examples show different ways to create a populated `List`:
 
+{% tabs collection_1 class=tabs-scala-version %}
```

Comment:
`class=tabs-scala-version` should not be used for "Scala 2 and 3" tabs. Soon I am adding assertions to make CI fail for this
```suggestion
{% tabs collection_1 %}
```